symengine/cwrapper.h

```diff
@@ -85,7 +85,11 @@ basic_struct* basic_new_heap();
 void basic_free_heap(basic_struct *s);
 
 //! Use these functions to get the commonly used constants as basic.
+
 //! Assigns to s a SymEngine constant with name c
+//! This function creates a new SymEngine::Constant from a copy of
+//! the string in c, thus the caller is free to use c afterwards,
+//! and also the caller must free c.
 void basic_const_set(basic s, char* c);
```

Comment:
@abinashmeher999, this is unrelated to the PR, but do we need this method in cwrappers? User can use a symbol instead of a constant.